packages/kbn-guided-onboarding/src/components/landing_page/guide_cards.constants.tsx

```diff
@@ -27,11 +27,13 @@ export interface GuideCardConstants {
   // see this issue https://github.com/elastic/kibana/issues/146672
   telemetryId: string;
   order: number;
+  icon: string;
 }
```

Comment:
just fyi I think the property `order` was planned to be used potentially for sorting and grouping. But it's probably not needed anymore.